Documentation/debugging/windows-instructions.md

```diff
@@ -75,7 +75,64 @@ For quick reference, type the following in WinDBG:
 
 ## Traces
 
-In Windows, EventSource generated traces are collected via ETW using PerfView.
+In Windows, EventSource generated traces are collected via ETW using either logman or PerfView.
+
+### Using Logman
+[Logman](https://technet.microsoft.com/en-us/library/bb490956.aspx) is built in within the Windows Operating and so doesn't need to be downloaded or installed.
```

Comment:
"is built in within the Windows Operating" -> "ships with Windows" is better phrasing.